presto-spi/src/main/java/com/facebook/presto/spi/connector/ConnectorMetadata.java

```diff
@@ -275,6 +277,14 @@ default void dropColumn(ConnectorSession session, ConnectorTableHandle tableHand
         return Optional.of(new ConnectorNewTableLayout(partitioningHandle, partitionColumns));
     }
 
+    /**
+     * Describes statistics that must be collected
```

Comment:
Nit: end Javadoc sentences in a period

Reply:
Let's make this say "during a write" for now. We can update to "during a write or analyze" later when that is implemented.